webpack/index.js

```diff
@@ -0,0 +1,39 @@
+import chalk from "chalk"
```

Comment:
No `@ts-check` here?

Reply:
Right now `@ts-check` doesn't like that we're coercing webpack's `mode` into a string (vs an enum value) and so it flags. Once this is stable i'm going to loop back convert these files to typescript but need to do a bit more setup.